docs/platform/what-is-tizen/versions/tizen-5-5-m2.md

```diff
@@ -0,0 +1,66 @@
+# Tizen 5.5
+
+[The second milestone (M2) release of Tizen 5.5](../../release-notes/tizen-5-5-m2.md) was announced in October 2019.
+Tizen is an open-source operating system (OS) maintained by Samsung and is developed and optimized for smart devices since 2012.
+Currently, more than 130+ million Tizen-based devices such as TVs, watches, Family-Hubs, IoT based home appliances, and so on are commercialised.
+
+The following Samsung devices are commercialised with Tizen OS:
+Samsung TVs (launched in 2015), Galaxy Wearables (launched in 2014), Family-Hub Refrigerators (launched in 2016), Smart Signage (launched in 2019), Air Conditioners (launched in 2019), Flip board (launched in 2019) and so on.
+
+The key features of Tizen 5.5 are the enhancement of development environment and core, supporting AI component and the expansion of IoT profile.
+
+## Enhancement of Development Environment and Core Component
+
+- **Tizen .NET** :
+It is the development environment for developing high quality and high performance application in C#. There is the enhancement in Tizen Development environment in Tizen 5.5.
+The latest Xamarin.Forms 4.0 and the latest .NET Core 3.0 are now supported. Xamarin.Forms on Tizen .NET is a complete cross-platform UI toolkit that allows you to create native UI. The latest .NET Core supports unloading DLL, AOT, pre-loading, pre-initialization,Tiered complications and MPGO for enhancing application launching time. Also, there is memory optimization by reducing relocation of DLL files, pre-loading for resource sharing and so on. The .NET Core 3.0 Runtime is faster, lighter than its previous version. With all these enhancements, the application launching time is reduced by 28% and memory usage is reduced by 20%. New Tizen extension for official visual studio for Mac also is released in Tizen 5.5. The key features of the Tizen extension are the Tizen.NET application development environment, the Tizen IDE tools, and the application debugging using .NET Core debugger.
```

Comment:
1. Line 16: Insert one space after pre-initialization, like this:  
**Must be Fixed like this:** pre-initialization, Tiered complications and MPGO for enhancing application launching time.

2. Line 16: Insert article before official, placement of also: 
**Must be Fixed like this:** New Tizen extension for the official visual studio for Mac is also released in Tizen 5.5.

3. Line 41: Remove s after allow:  

    **Must be Fixed like this:** ML Inference API sets allow application developers to apply their machine learning mechanisms or neural network models easily and efficiently.

4. Line 43: Replace have with has 

     **Must be Fixed like this:** The ML API set has two API subsets, Pipeline ML API and Single ML API. 
  ML API sets of the current version of Tizen support machine learning inferences only. 

5. Line: 57 
  Replace via to by 
  Insert comma after API 
  Insert that 
  **Current:** 
  In Tizen 5.5, you can recognize the face and facial landmark via calling MediaVision API is based on 
  AI Framework such as Caffe and TF-Lite.
  **Must be Fixed like this:**
  In Tizen 5.5, you can recognize the face and facial landmark by calling MediaVision API, that is based 
  on AI Frameworks such as Caffe and TF-Lite.

Reply:
I raised the commit based on you comment. I hope this will be the final one. Thanks for detail feedback.

Reply:
@ just one comment missed. 

copy paste this at line 43: 

ML API sets are based on [NNStreamer](https://github.com/nnsuite/nnstreamer) that allows construction of the GStreamer pipelines that consists of the conventional GStreamer media plugins and the neural network plugins. There are plans to expand its functionality to on-device training in the later releases of Tizen. The ML API set has two API subsets, Pipeline ML API and Single ML API. ML API sets of the current version of Tizen support machine learning inferences only.

Reply:
I did with new commit.